### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by manishitMani**
updated some docs

Fixes #<Replace with the number of the issue, Mandatory>

One sentence summary of the change.

## Proposed Changes (Mandatory)

A brief list of proposed changes in order to fix the issue:

  -
  -
  -

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -3067,7 +3067,7 @@ <h4 id="_load_jdbc_resources"><a class="link" href="#_load_jdbc_resources">LOAD
 *</p>
 </li>
 <li>
-<p><a href="https://neo4j.com/blog/apoc-database-integration-import-export-cypher/">APOC: Database Integration, Import and Export with Awesome Procedures On Cypher</a></p>
+<p><a href="https://neo4j.com/blog/apoc-database-integration-import-export-cypher/">APOC: Database Integration, Import and Export with Awesome Procedures available on Cypher</a></p>
```
Comment (gem-neo4j) [inline docs/index.html:3070]: It is Awesome Procedures on Cypher because that is what APOC stands for :)

File: docs/index.html [hunk 2]
```diff
@@ -3292,8 +3292,8 @@ <h4 id="_load_xml_introduction"><a class="link" href="#_load_xml_introduction">L
 <p>Many existing (enterprise) applications, endpoints and files use XML as data exchange format.</p>
 </div>
 <div class="paragraph">
-<p>To make these datastructures available to Cypher, you can use <code>apoc.load.xml</code>.
-It takes a file or http URL and parses the XML into a map datastructure.</p>
+<p>To make these Data Structures available to Cypher, you can use <code>apoc.load.xml</code>.
```
Comment (gem-neo4j) [inline docs/index.html:3295]: We don't tend to capitalise words like this in our docs, so it would need to be: data structures

Comment (manishitMani) [reply docs/index.html:3295]: ok.. thanks

**Comment by gem-neo4j** [comment]
I'll close this as nothing as been updated in a while :)

**Pull request closed without merging**